docs/guides/platform/get-started/getting-started-with-linode/index.md

```diff
@@ -61,10 +62,14 @@ Use a [StackScript](http://www.linode.com/stackscripts) to quickly deploy softwa
 
 1. Select a Linode plan.
 
+![Getting Started Create Linode](getting-started-create1.png "Getting Started Create Linode")
```

Comment:
Please indent the image because the steps are getting renumbered:https://deploy-preview-3873--nostalgic-ptolemy-b01ab8.netlify.app/docs/guides/getting-started/